### PR TITLE
feat: (home page) Structure with ARIA labels.

### PR DESCRIPTION
**Pull request by saintograph**
Images and dummy text for the home page. ARIA roles and labels added to each semantic tag.

### CODE REVIEW

**Comment by saintograph** [comment]
@adityamukho IDs added to each main section.

File: index.html [hunk 1]
```diff
@@ -23,7 +23,44 @@
...
+  </section>
+  <section id="attractions-main" role="contentinfo" aria-label="content about theme park rides and facilities">
+    <div role="link" id="attractionsSection">
+      <h3>Restaurants</h3>
+      <img src="./img/restaurants.jpg" alt="theme park restaurants"/>
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam non ex sed mi pulvinar iaculis a ut metus. Etiam aliquam hendrerit posuere. Proin molestie risus a arcu commodo volutpat.
+      </p>
+    </div>
+    <div role="link" id="attractionsSection">
```
Comment (adityamukho) [inline index.html]: Duplicate id. I think it should suffice to id the outer section elements.

File: index.html [hunk 2]
```diff
@@ -23,7 +23,44 @@
...
+      <h3>Rides</h3>
+      <img src="./img/rides.jpg" alt="theme park rides"/>
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam non ex sed mi pulvinar iaculis a ut metus. Etiam aliquam hendrerit posuere. Proin molestie risus a arcu commodo volutpat.
+      </p>
+    </div>
+    <div role="link" id="attractionsSection">
```
Comment (adityamukho) [inline index.html]: Same as above. Duplicate id.

File: index.html [hunk 3]
```diff
@@ -23,7 +23,44 @@
...
+      <h3>Facilities</h3>
+      <img src="./img/facilities.jpg" alt="theme park facilities"/>
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam non ex sed mi pulvinar iaculis a ut metus. Etiam aliquam hendrerit posuere. Proin molestie risus a arcu commodo volutpat.
+      </p>
+    </div>
+    <div role="link" id="attractionsSection">
```
Comment (adityamukho) [inline index.html]: ^^

**Comment by saintograph** [comment]
1. Removed duplicate IDs,
2. Created attractions page ( i.e. attractions.html ),
3. Added html-include script,

**Pull request merged**